ToyBox/classes/MonkeyPatchin/BagOfPatches/MiscWrath.cs

```diff
@@ -675,6 +677,29 @@ private static void Postfix(Polymorph __instance) {
                 __instance.Owner.View.transform.localScale = new Vector3(scale, scale, scale);
             }
         }
+        [HarmonyPatch(typeof(LocalizedString))]
+        internal class BPTagger 
+        {
+            private static LocalizationPack pack = Kingmaker.Localization.LocalizationManager.LoadPack(Kingmaker.Localization.LocalizationManager.s_CurrentLocale);
+
+            [HarmonyPatch(nameof(LocalizedString.LoadString)), HarmonyPostfix]
+            public static void LoadString_ModTagPatch(ref string __result, LocalizedString __instance) 
+            {
+                if (settings.togglemoddedbptag) 
+                {
+                    try 
+                    {
+                        if (!__result.Contains(".") && !__result.Contains("。")) { return; }
```

Comment:
What are those strings? A comment to mention that they are a specific kind (in which the tag is not wanted) might be useful.

Reply:
No idea, just stole from String

Reply:
I just pinged him about it so whatever